cmd/bootstrap/runner.go

```diff
@@ -272,36 +284,24 @@ func (r *runner) installAppCatalogs(ctx context.Context, k8sClients k8sclient.In
 	return nil
 }
 
-func (r *runner) installChartMuseum(ctx context.Context, k8sClients k8sclient.Interface) error {
+func (r *runner) installChartMuseum(ctx context.Context, appTest apptest.Interface) error {
 	var err error
 
 	{
 		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("creating %#q app cr", chartMuseumName))
 
-		appCR := &v1alpha1.App{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      chartMuseumName,
-				Namespace: namespace,
-				Labels: map[string]string{
-					// Processed by app-operator-unique.
-					label.AppOperatorVersion: "0.0.0",
-				},
-			},
-			Spec: v1alpha1.AppSpec{
-				Catalog: helmStableCatalogName,
-				KubeConfig: v1alpha1.AppSpecKubeConfig{
-					InCluster: true,
-				},
-				Name:      chartMuseumName,
-				Namespace: namespace,
-				Version:   chartMuseumVersion,
+		apps := []apptest.App{
+			{
+				CatalogName:   helmStableCatalogName,
+				CatalogURL:    helmStableCatalogStorageURL,
+				Name:          chartMuseumName,
+				Namespace:     namespace,
+				Version:       chartMuseumVersion,
+				WaitForDeploy: false,
```

Comment:
```suggestion
```
It should be`false` as default, should we keep it?

Reply:
Yes we can simplify. The flag is for using in integration tests when you want to wait for the app to be installed before starting the test.